helpdesk-bot/src/test/resources/jbehave.stories/helpdeskbot_ticket_management.story

```diff
@@ -0,0 +1,74 @@
+Meta: @helpdeskbot
+
+Narrative:
+As a user
+I want to perform a ticket management with bot
+So that I can achieve a business goal
+
+!-- Create ticket scenarios
+
+Scenario: Create a chat (ticket) with helpdeskbot
+Given a new user helpdesk_super_user in a private pod with roles INDIVIDUAL,SUPER_ADMINISTRATOR
+!--And a certificate for API user helpdesk_super_user
+!--When API user helpdesk_super_user authenticates using a certificate
+When user helpdesk_super_user logs in
+And set bot user helpdeskbot
+And user helpdesk_super_user creates an IM with helpdeskbot
+And helpdeskbot create a ticket to help the client helpdesk_super_user
+Then helpdeskbot send a successful message to helpdesk_super_user
+
+Scenario: Try to create new ticket without close other before
+Then helpdesk_super_user talk to helpdeskbot
+And helpdeskbot send a message to ticket room without create new ticket
+And helpdeskbot not send a successful message to helpdesk_super_user
+
```

Comment:
I believe that the scenario is missing when there are 2 or more agents in the room, and one of the agents claim the ticket, after this
 the bot sends the message that the ticket was claimed by user1, and for agent 2 the button claim is hidden automatically.

Reply:
Good.. I will add this scenario in the story